src/cms-content/recommendations/index.ts

```diff
@@ -26,15 +26,30 @@ export enum HarvardLevel {
 export enum RecommendCategory {
   MASKS = 'MASKS',
   GATHERINGS = 'GATHERINGS',
-  SCHOOLS = 'SCHOOLS',
   RESTAURANTS = 'RESTAURANTS',
+  BARS = 'BARS',
+  MEDICAL_CONDITIONS = 'MEDICAL_CONDITIONS',
+  PUBLIC_INTERACTIONS = 'PUBLIC_INTERACTIONS',
+  SCHOOLS = 'SCHOOLS',
 }
 
 export enum RecommendID {
-  MASKS_INDOORS = 'masks-indoors',
-  GATHERINGS_OUTDOORS = 'gatherings-outdoors',
-  SCHOOLS_ORANGE = 'schools-orange',
-  RESTAURANTS_25_PERCENT = 'restaurants-indoor-25-percent',
+  MASKS_YELLOW = 'MASKS_YELLOW',
+  GATHERINGS_YELLOW = 'GATHERINGS_YELLOW',
+  BARS_YELLOW = 'BARS_YELLOW',
+  RESTAURANTS_YELLOW = 'RESTAURANTS_YELLOW',
+  MEDICAL_CONDITIONS_YELLOW = 'MEDICAL_CONDITIONS_YELLOW',
+  PUBLIC_INTERACTIONS_YELLOW = 'PUBLIC_INTERACTIONS_YELLOW',
+  PUBLIC_INTERACTIONS_RED = 'PUBLIC_INTERACTIONS_RED',
+  MEDICAL_CONDITIONS_RED = 'MEDICAL_CONDITIONS_RED',
+  RESTAURANTS_RED = 'RESTAURANTS_RED',
+  BARS_RED = 'BARS_RED',
+  GATHERINGS_RED = 'GATHERINGS_RED',
+  MASKS_RED = 'MASKS_RED',
+  SCHOOLS_RED = 'SCHOOLS_RED',
+  SCHOOLS_ORANGE = 'SCHOOLS_ORANGE',
+  SCHOOLS_YELLOW = 'SCHOOLS_YELLOW',
+  SCHOOLS_GREEN = 'SCHOOLS_GREEN',
```

Comment:
this ID was originally intended to filter for sub-levels. if its just a combo of category+level, do we need it?

Reply:
Let's keep them a bit longer just in case. The sub-level was actually a recommendation for public officials, but we might want to tweak it so it's part of public messaging